### PR TITLE
feat: [M3-7029] - Add AGLB Certificate Create Drawer

### PR DESCRIPTION
**Pull request by bnussman-akamai**
## Description 📝

- Adds Create Certificate Drawer ✨

## Preview 📷
Screenshot 2023-08-31 at 2 34 38 PM

## How to test 🧪
- Go to `http://localhost:3000/loadbalancers/1/certificates` with the **MSW on** and try to upload a certificate (you can use fake values)
- Nothing will really happen when you submit the form successfully because of the inherent nature of React Query and the MSW
- Check client side validation

### CODE REVIEW

**Review by abailly-akamai — APPROVED** [review]
👍 looks great!

File: packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Certificates/CreateCertificateDrawer.tsx [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+          value={formik.values.certificate}
+        />
+        <TextField
+          errorText={errorMap.key}
+          label="Private Key"
+          labelTooltipText="TODO"
+          multiline
+          name="key"
+          onChange={formik.handleChange}
+          value={formik.values.key}
```
Comment (abailly-akamai) [inline packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Certificates/CreateCertificateDrawer.tsx:120]: You may want to add the new `trimmed` prop on both text areas

**Review by abailly-akamai — APPROVED** [review]
Looks great 👍 

Is there any type of additional validation we should run on those fields? Not sure if there's an expected shape for either

File: packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Certificates/CreateCertificateDrawer.tsx (hunk 1 above)
Comment (bnussman-akamai) [reply packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Certificates/CreateCertificateDrawer.tsx:120]: Good call

File: packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Certificates/CreateCertificateDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+          errorText={errorMap.label}
+          label="Label"
+          name="label"
+          onChange={formik.handleChange}
+          value={formik.values.label}
+        />
+        <TextField
+          errorText={errorMap.certificate}
+          label="TLS Certificate"
+          labelTooltipText="TODO"
```
Comment (jaalah-akamai) [inline packages/manager/src/features/LoadBalancers/LoadBalancerDetail/Certificates/CreateCertificateDrawer.tsx:107]: Just making sure we have this tracked to complete in the future?

**Review by jaalah-akamai — APPROVED** [review]

**Pull request merged**